packages/driver/src/multi-domain/domain_fn.ts

```diff
@@ -60,6 +60,8 @@ export const handleDomainFn = (Cypress: Cypress.Cypress, cy: $Cy) => {
     const stateUpdates = {
       ...state,
       redirectionCount: {}, // This is fine to set to an empty object, we want to refresh this count on each switchToDomain command.
+      window: undefined, // reset the window and document, these will be reset when the AUT loads.
```

Comment:
With this change, I believe this try...catch can be removed.
https://github.com/cypress-io/cypress/blob/feature-multidomain/packages/driver/src/cy/commands/navigation.ts#L996

Reply:
Looks like you're correct! Happy to remove these.